src/OrchardCore/OrchardCore.Lucene.Core/QueryProviders/SimpleQueryStringQueryProvider.cs

```diff
@@ -14,9 +15,19 @@ public Query CreateQuery(ILuceneQueryService builder, LuceneQueryContext context
             }
 
             var queryString = query["query"]?.Value<string>();
-            var fields = query["fields"]?.Value<string>() ?? "";
-            var defaultOperator = query["default_operator"]?.Value<string>() ?? "and";
```

Comment:
@sebastienros asked: I would prefer not to change the default. I assume that the more terms you add, the less results you get. Unless the Elastic search states the opposite.

Reply:
Elastic Search does state the opposite. According to https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-simple-query-string-query.html, the default value for default_operator is OR.